integration_testing/features/coach/coach-lesson-difficult-questions.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Coach reviews difficult questions
+  Coach needs to be able to see which questions in exercises are difficult for the class learners
```

Comment:
This is not currently implemented, I am doing so now, hoping to get in for the beta.